src/Microsoft.Health.Fhir.TaskManagement/ITask.cs

```diff
@@ -0,0 +1,21 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Health.Fhir.TaskManagement
+{
+    public interface ITask : IDisposable
```

Comment:
Lets add docs for the interfaces to help in future

Reply:
Sure, added.